src/i_mbfsound.c

```diff
@@ -0,0 +1,145 @@
+//
+// Copyright (C) 1999 by
+// id Software, Chi Hoang, Lee Killough, Jim Flynn, Rand Phares, Ty Halderman
+// Copyright(C) 2023 ceski
```

Comment:
Please carry over the copyright from `i_sound.c` as well.

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1118/commits/08f5e783233d79c120f5d5db7c0a685b6e99e3ba